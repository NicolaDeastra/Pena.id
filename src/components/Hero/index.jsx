import * as React from 'react';
import { Heading, Text, VStack, Button, Center } from '@chakra-ui/react';

import siteConfig from '../../site-config';

const Hero = () => {
  return (
    <Center
      h="lg"
      pt={['0rem', '0rem']}
      align="center"
      flexDirection={{ base: 'column', md: 'row' }}
      mb={{ base: 20, md: '7rem' }}
    >
      <VStack spacing={6} px="10rem">
        <Heading as="h2" size="xl" mb="1.4rem" textAlign="center">
          {siteConfig.heading}
        </Heading>
        <Text fontSize={{ base: 'sm', md: 'md' }} textAlign="center">
          {siteConfig.subHeading}
        </Text>
        <Button size="lg" ml={['4.5rem', '0']} mt="24px">
          Get Started
        </Button>
      </VStack>
    </Center>
  );
};

export default Hero;
