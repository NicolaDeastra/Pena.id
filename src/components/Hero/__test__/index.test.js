import { render } from '@testing-library/react';
import Hero from '..';

test('Hero render correctly', () => {
  const { container } = render(<Hero />);
  expect(container).toMatchSnapshot();
});
