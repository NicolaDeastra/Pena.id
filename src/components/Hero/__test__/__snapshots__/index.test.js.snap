// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero render correctly 1`] = `
<div>
  <div
    align="center"
    class="css-1i0j96w"
  >
    <div
      class="chakra-stack css-xjf8f"
    >
      <h2
        class="chakra-heading css-ue84p2"
      >
        Publish Platform for Professional Bloggers
      </h2>
      <p
        class="chakra-text css-ecbqlw"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam tincidunt porta neque, ac ornare justo.
      </p>
      <button
        class="chakra-button css-1bhjgw4"
        type="button"
      >
        Get Started
      </button>
    </div>
  </div>
</div>
`;
